include/odp/api/spec/classification.h

```diff
@@ -253,6 +302,10 @@ typedef struct odp_cls_capability_t {
 	/** Maximum number of CoS supported */
 	unsigned int max_cos;
 
+	/** Maximum number of CoSes that can have statistics enabled at the same
+	 * time. */
+	uint32_t max_cos_stats;
```

Comment:
This could note in some form that: CoS level statistics are not supported when zero